### PR TITLE
Sovrn Bid Adapter: change id and size payload

### PR DESCRIPTION
**Pull request by TWrightSovrn**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/sovrnBidAdapter.js [hunk 1]
```diff
@@ -167,20 +154,29 @@ export const spec = {
...
         deepSetValue(sovrnBidReq, 'user.ext.eids', eids)
-        deepSetValue(sovrnBidReq, 'user.ext.tpid', tpid)
         if (criteoId) {
           deepSetValue(sovrnBidReq, 'user.ext.prebid_criteoid', criteoId)
         }
       }
 
-      let url = `https://ap.lijit.com/rtb/bid?src=$$REPO_AND_VERSION$$`;
-      if (iv) url += `&iv=${iv}`;
+      const urlParams = new URLSearchParams(window.location.search)
```
Comment (TWrightSovrn) [inline modules/sovrnBidAdapter.js:162]: This is only here to try to test the code, I will remove once I can test right.

File: modules/sovrnBidAdapter.js [hunk 2]
```diff
@@ -74,22 +74,9 @@ export const spec = {
       let iv;
       let schain;
       let eids;
-      let tpid = []
       let criteoId;
 
       _each(bidReqs, function (bid) {
-        if (!eids && bid.userId) {
```
Comment (TWrightSovrn) [inline modules/sovrnBidAdapter.js:81]: This is the TPIDS changes

File: modules/sovrnBidAdapter.js [hunk 3]
```diff
@@ -268,11 +264,16 @@ export const spec = {
...
 function _buildVideoRequestObj(bid) {
   const videoObj = {}
+  const bidSizes = deepAccess(bid, 'sizes')
   const videoAdUnitParams = deepAccess(bid, 'mediaTypes.video', {})
   const videoBidderParams = deepAccess(bid, 'params.video', {})
   const computedParams = {}
-
-  if (Array.isArray(videoAdUnitParams.playerSize)) {
+  
+  if (bidSizes) {
```
Comment (TWrightSovrn) [inline modules/sovrnBidAdapter.js:272]: This is the changes for the size stuff, no clue if this one is right, might need more clarification.

File: modules/sovrnBidAdapter.js [hunk 4]
```diff
@@ -167,20 +154,29 @@ export const spec = {
...
           deepSetValue(sovrnBidReq, 'user.ext.prebid_criteoid', criteoId)
         }
       }
 
-      let url = `https://ap.lijit.com/rtb/bid?src=$$REPO_AND_VERSION$$`;
-      if (iv) url += `&iv=${iv}`;
+      const urlParams = new URLSearchParams(window.location.search)
+      const uuid = urlParams.get('testUuid')
+
+      let url = `https://localhost:8090/rtb/prebidResponse1/bid/openrtb25`
```
Comment (patmmccann) [inline modules/sovrnBidAdapter.js:165]: is this correct?

**Comment by TWrightSovrn** [comment]
Wrong thing

File: modules/sovrnBidAdapter.js (hunk 4 above)
Comment (TWrightSovrn) [reply modules/sovrnBidAdapter.js:165]: Ignore this, I did a PR wrong.

**Pull request closed without merging**